Assets/Editor/AmplitudeDependencies.xml

```diff
@@ -4,7 +4,7 @@
       <repository>https://repo.maven.apache.org/maven2</repository>
     </repositories>
 
-    <androidPackage spec="com.amplitude:android-sdk:2.38.3">
+    <androidPackage spec="com.amplitude:android-sdk:2.40.+">
```

Comment:
Any risk making this a range? I'm assuming not as the aar is embedded but just want to double check.

Reply:
This is mostly done so that we don't have to deal with updating this SDK every time the underlying version changes. If customers use the package with the dependencies, then those are locked at the version that we check in. But if they use the package without dependencies and rely on their local dependency pull, then it will pick the latest underlying version.